content/docs/components-and-props.md

```diff
@@ -78,24 +85,33 @@ ReactDOM.render(
 
 [](codepen://components-and-props/rendering-a-component)
 
-Let's recap what happens in this example:
+Mari merangkum apa yang terjadi pada contoh kali ini:
 
-1. We call `ReactDOM.render()` with the `<Welcome name="Sara" />` element.
-2. React calls the `Welcome` component with `{name: 'Sara'}` as the props.
-3. Our `Welcome` component returns a `<h1>Hello, Sara</h1>` element as the result.
-4. React DOM efficiently updates the DOM to match `<h1>Hello, Sara</h1>`.
+1. Kita memanggil `ReactDOM.render()` dengan elemen `<Welcome name="Sara" />`.
+2. React memanggil komponen `Welcome` dengan `{name: 'Sara'}` sebagai
+   propertinya.
+3. Komponen `Welcome` kita akan mengembalikan sebuah element `<h1>Hello,
+   Sara</h1` sebagai hasilnya.
+4. React DOM dengan efisien akan mengupdate DOM yang sesuai `<h1>Hello, Sara`.
 
->**Note:** Always start component names with a capital letter.
+>**Catatan:** Selalu awali nama komponen dengan sebuah huruf kapital.
 >
->React treats components starting with lowercase letters as DOM tags. For example, `<div />` represents an HTML div tag, but `<Welcome />` represents a component and requires `Welcome` to be in scope.
+>React memperlakukan awalan komponen dengan huruf kecil sebagai tag dari DOM.
+>Sebagai contoh, `<div />`, merepresentasikan sebuah HTML div tag, tetapi
+>`<Welcome />` merepresentasikan sebuah komponen dan membutuhkan Welcome to be
+>in scope.
 >
->You can read more about the reasoning behind this convention [here.](/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized)
+>Anda dapat membaca lebih lanjut tentang alasan dibalik Konvensi [disini.](/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized)
```

Comment:
Anda dapat membaca lebih lanjut tentang alasan dibalik konvensi tersebut [disini]

What do you think? 😄